Automattic/wp-calypso
# Shadow Help Center styles in wp-admin and Gutenberg

## Proposed Changes

This wraps the Help Center in a shadow root to prevent styles from leaking in and out of the Help Center.

## Why are these changes being made?
The Help Center styles can break a few things and be broken by a few things because the Help Center runs in many places under several environments.

## Testing Instructions

1. In `apps/help-center` run `yarn dev --sync`.
2. Test in wp-admin/post-new.php and wp-admin.
3. Make sure every route of the Help Center is well-styled. 

## Next

Once this is live. We can merge [this](https://github.com/Automattic/jetpack/pull/38935)
